### PR TITLE
adding deeptime recipe

### PR DESCRIPTION
**Pull request by cwehmeyer**
This is a dummy package which will soon be replaced by the content of github.com/markovmodel/deeptime.

### CODE REVIEW

File: recipes/deeptime/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    - python
+
+test:
+  imports:
+    - deeptime
+
+about:
+  home: http://github.com/markovmodel/deeptime
+  license: LGPL-3.0
+  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE'
```
Comment (CJ-Wright) [inline recipes/deeptime/meta.yaml]: Please include license family

File: recipes/deeptime/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+requirements:
+  build:
+    - python
+    - pip
+  run:
+    - python
+
+test:
+  imports:
+    - deeptime
```
Comment (CJ-Wright) [inline recipes/deeptime/meta.yaml]: Please include more tests (more import tests are okay)

**Comment by marscher** [comment]
LGTM

File: recipes/deeptime/meta.yaml (hunk 2 above)
Comment (marscher) [reply recipes/deeptime/meta.yaml]: as @cwehmeyer said, this is atm just a placeholder, so there are no more useful tests possible than just importing this.

Comment (CJ-Wright) [reply recipes/deeptime/meta.yaml]: I'm confused, is this code ready to be packaged?

**Comment by marscher** [comment]
It is IMHO, we expect an update with real content pretty soon. I've
discussed this on conda-forges gitter some time ago, if it is possible
to reserve a package name by adding a skeleton and we concluded that we
should just PR it.

Thank you for reviewing this.

**Comment by marscher** [comment]
screenshot_20180212_172005

File: recipes/deeptime/meta.yaml (hunk 2 above)
Comment (cwehmeyer) [reply recipes/deeptime/meta.yaml]: Yes, it is. It is in a very early state but that should, as shown below, not be an issue.

**Comment by synapticarbors** [comment]
Just checking in on the status of this PR. Is this mergeable?

PS - @cwehmeyer: congrats on the JCP paper related to this package. It just popped up on my Google Scholar recommendations.

**Comment by marscher** [comment]
@synapticarbors thanks to almighty google we can finally grab the package name :) This is mergable - it will just place a dummy. But expect the final/usable stuff pretty soon.

**Comment by marscher** [comment]
@franknoe fyi

**Comment by marscher** [comment]
@amardt @Obudo I guess you want to assure that this gets merged someday?

**Pull request closed without merging**